Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -194,26 +258,33 @@ function updateReslice(
     keepFocalPointPosition: false, // Defines if the focal point position is kepts (same display distance from reslice cursor center)
     computeFocalPointOffset: false, // Defines if the display offset between reslice center and focal point has to be
     // computed. If so, then this offset will be used to keep the focal point position during rotation.
+    spheres: null,
+    resetViewUp: false, // Defines if the camera view up is projected on plane (resetViewUp = false) or if we use the image bounds (resetViewUp = true)
   }
 ) {
-  const modified = widget.updateReslicePlane(
+  const obj = widget.updateReslicePlane(
     interactionContext.reslice,
     interactionContext.viewType
   );
-  if (modified) {
+  if (obj.modified) {
     // Get returned modified from setter to know if we have to render
     interactionContext.actor.setUserMatrix(
       interactionContext.reslice.getResliceAxes()
     );
+    interactionContext.sphereSources[0].setCenter(...obj.origin);
```

Comment:
interactionContext.sphereSources.forEach(...)

Reply:
Not sure how to use the foreach as each point has a different value ?

Reply:
Indeed. Nevermind then